src/components/WwwFrame/WwwPage.vue

```diff
@@ -56,10 +64,29 @@ export default {
 	data() {
 		return {
 			isKivaAppReferral: false,
+			isNavUpdateExp: false,
 		};
 	},
 	created() {
 		this.isKivaAppReferral = this.$route?.query?.kivaAppReferral === 'true';
+
+		try {
+			const navUpdateExp = this.apollo.readQuery({
+				query: experimentAssignmentQuery.default,
+				variables: { id: NAV_UPDATE_EXP_KEY }
+			});
+			this.isNavUpdateExp = navUpdateExp?.experiment?.version === 'b';
+
+			trackExperimentVersion(
+				this.apollo,
+				this.$kvTrackEvent,
+				'Lending',
+				'kiva_nav_update',
+				'EXP-MP-1817-Jul2025'
```

Comment:
Looks like this might be the wrong ticket. Probably can change this to `EXP-MP-1696-Aug2025`